portal/migrations/0002_eventsobject.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 5.0.2 on 2024-09-11 15:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("portal", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="EventsObject",
```

Comment:
Just as an idea, maybe we should call it `EventsNews`. Just a tiny concern that it's a terminology overload between technical and news terms.

But we could just ignore that

Reply:
Ah, that's a good point. I am worried about mixing it up with the news. I am open to changing it, let's discuss it a bit before I make my second planned PR regarding event announcements in the near future.